CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2191,7 +2191,7 @@ protected override void OnMouseLeave(MouseEventArgs e)
         /// <param name="e">The <see cref="MouseButtonEventArgs"/> instance containing the event data.</param>
         private void OnMouseButton(MouseButtonEventArgs e)
         {
-            if (!e.Handled && browser != null)
+            if (!e.Handled && e.StylusDevice == null && browser != null)
```

Comment:
What is the reason for the additional check? Please add a comment explaining.

Reply:
Added comment in code. (Hopefully that's where you meant)